### PR TITLE
Provide descriptive titles to links in location_changes.md

### LINKED ISSUE #2213: Links do not include descriptive title in location_changes.md
### Steps to recreate
Visit docs/location_changes.md

### URL
[Location Changes document](https://github.com/pulibrary/bibdata/blob/main/docs/location_changes.md).

### Issues
Links do not include descriptive title in location_changes.md

### Severity
Low

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Harder for users with screen readers do determine purpose of links

### Recommended fix
Provide descriptive title for links

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes

### PR DESCRIPTION
**Pull request by lizgarciao**
closes #2213

### CODE REVIEW

File: docs/location_changes.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # When a location changes in Alma:
-* A ticket should be created first in https://github.com/PrincetonUniversityLibrary/alma-config/issues with all the necessary information.
+* A ticket should be created first in [issues](https://github.com/PrincetonUniversityLibrary/alma-config/issues) with all the necessary information.
```
Comment (christinach) [inline docs/location_changes.md:2]: can we update the link text to -> [Alma Config repository]

File: docs/location_changes.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
 6. Load locally the rails server; Go to: `localhost:<portnumber>/locations/holding_locations` in your browser and make sure that the locations have been updated.
 
 ## 2. Update Bibdata staging
-Test the updated locations in Bibdata-staging https://bibdata-staging.princeton.edu/
+Test the updated locations in [Bibdata-staging](https://bibdata-staging.princeton.edu/)
```
Comment (christinach) [inline docs/location_changes.md:23]: Can you change the bibdata-staging link text to [Bibdata staging] ?

File: docs/location_changes.md [hunk 3]
```diff
@@ -30,10 +30,10 @@ Deploy your branch on staging and run the following steps to make sure that noth
 2. Run the following rake task to delete and repopulate the locations in the bibdata staging database:  
   `RAILS_ENV=production bundle exec rake bibdata:delete_and_repopulate_locations`
 
-3. Review the changes in https://bibdata-staging.princeton.edu/
+3. Review the changes in [Bibdata-staging](https://bibdata-staging.princeton.edu/).
```
Comment (christinach) [inline docs/location_changes.md:33]: Can you change the bibdata-staging link text to [Bibdata staging] ?

File: docs/location_changes.md [hunk 4]
```diff
@@ -30,10 +30,10 @@ Deploy your branch on staging and run the following steps to make sure that noth
...
 
 ## 3. Update Bibdata qa
-Test the updated locations in Bibdata-qa https://bibdata-qa.princeton.edu/
+Test the updated locations in [Bibdata-qa](https://bibdata-qa.princeton.edu/).
```
Comment (christinach) [inline docs/location_changes.md:36]: Can you change the bibdata-qa link text to [Bibdata qa] ?

File: docs/location_changes.md [hunk 5]
```diff
@@ -43,11 +43,11 @@ Deploy your branch on qa and run the following steps to make sure that nothing i
 2. Run the following rake task to delete and repopulate the locations in the bibdata qa database:  
   `RAILS_ENV=production bundle exec rake bibdata:delete_and_repopulate_locations`
 
-3. Review the changes in https://bibdata-qa.princeton.edu/
+3. Review the changes in [Bibdata-qa](https://bibdata-qa.princeton.edu/)
```
Comment (christinach) [inline docs/location_changes.md:46]: Can you change the bibdata-qa link text to [Bibdata qa] ?

File: docs/location_changes.md [hunk 6]
```diff
@@ -43,11 +43,11 @@ Deploy your branch on qa and run the following steps to make sure that nothing i
...
 
 *If it runs successfully merge and deploy to production; go to the next step to update the location tables in production.*
 ## 4. Update Bibdata production
-### Don't update the locations in Bibdata production during indexing hours https://github.com/pulibrary/bibdata/blob/main/docs/alma_publishing_jobs_schedule.md.
+### Don't update the locations in Bibdata production during indexing hours. Please see [schedule](https://github.com/pulibrary/bibdata/blob/main/docs/alma_publishing_jobs_schedule.md).
```
Comment (christinach) [inline docs/location_changes.md:50]: can we update the link to [Alma Publishing jobs schedule]?

**Review by christinach — CHANGES_REQUESTED** [review]
Thank you @lizgarciao. I added a few comments/changes

**Comment by lizgarciao** [comment]
> Thank you @lizgarciao. I added a few comments/changes

I have included your suggestions.

**Review by christinach — APPROVED** [review]

**Pull request merged**